marlowe/src/Language/Marlowe/Client.hs

```diff
@@ -128,7 +133,7 @@ data PartyAction
              | CloseContract
   deriving (Show)
 
-type RoleOwners = AssocMap.Map Val.TokenName PubKeyHash
+type RoleOwners = AssocMap.Map Val.TokenName (AddressInEra ShelleyEra)
```

Comment:
`AddressInEra ShelleyEra` should work. We may want to adjust this eventually, when for instance we allow scripts to hold role tokens.

Reply:
`AddressInEra` has a `FromJSON` instance, `AddressAny` doesn't